lib/Newsletter/ViewInBrowser/ViewInBrowserRenderer.php

```diff
@@ -48,11 +43,16 @@ public function render(
     bool $isPreview,
     Newsletter $newsletter,
     SubscriberEntity $subscriber = null,
-    SendingQueue $queue = null
+    SendingQueueEntity $queue = null
   ) {
     $wpUserPreview = $isPreview;
     if ($queue && $queue->getNewsletterRenderedBody()) {
-      $newsletterBody = $queue->getNewsletterRenderedBody('html');
+      $body = $queue->getNewsletterRenderedBody();
+      if (is_array($body)) {
+        $newsletterBody = $body['html'];
+      } else {
+        $newsletterBody = '';
```

Comment:
I looked at the old implementation of the getter `getNewsletterRenderedBody`, and I think you made a mistake in the else statement. I guess that there should be `$newsletterBody = $body`. Or is there some reason for the empty string?

Reply:
Basically, I did it because PHPStan was yelling at me. I would say that normally this would never be used. When a user displays a newsletter in a browser the `$body['html']` should always be set. If it is not something really went wrong so I guess the empty string is ok in this case.